src/Symfony/Component/Asset/Packages.php

```diff
@@ -38,6 +38,9 @@ public function __construct(PackageInterface $defaultPackage = null, array $pack
         }
     }
 
+    /**
+     * @param PackageInterface $defaultPackage
```

Comment:
This one can be removed as all the information is already contained in the method signature.